### PR TITLE
Add ingress config next step to OSP install docs

### PR DESCRIPTION
**Pull request by maxwelldb**
Context: https://bugzilla.redhat.com/show_bug.cgi?id=1850999

### CODE REVIEW

**Comment by maxwelldb** [comment]
QE :+1:: https://bugzilla.redhat.com/show_bug.cgi?id=1850999#c18

File: installing/installing_openstack/installing-openstack-installer-kuryr.adoc [hunk 1]
```diff
@@ -51,3 +51,4 @@ include::modules/installation-osp-configuring-floating-ip.adoc[leveloffset=+1]
 * xref:../../installing/install_config/customizations.adoc#customizations[Customize your cluster].
 * If necessary, you can
 xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[opt out of remote health reporting].
+* If you need to enable external access to node ports, xref:../../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-nodeport.html#nw-using-nodeport_configuring-ingress-cluster-traffic-nodeport[configure ingress cluster traffic by using a NodePort].
```
Comment (bobfuru) [inline installing/installing_openstack/installing-openstack-installer-kuryr.adoc:54]: s/.htmtl/.adoc/

File: installing/installing_openstack/installing-openstack-user-kuryr.adoc [hunk 2]
```diff
@@ -65,4 +65,5 @@ include::modules/installation-osp-configuring-floating-ip.adoc[leveloffset=+1]
 
 * xref:../../installing/install_config/customizations.adoc#customizations[Customize your cluster].
 * If necessary, you can
-xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[opt out of remote health reporting].
\ No newline at end of file
+xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[opt out of remote health reporting].
+* If you need to enable external access to node ports, xref:../../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-nodeport.html#nw-using-nodeport_configuring-ingress-cluster-traffic-nodeport[configure ingress cluster traffic by using a NodePort].
```
Comment (bobfuru) [inline installing/installing_openstack/installing-openstack-user-kuryr.adoc:69]: s/.htmtl/.adoc/

**Comment by maxwelldb** [comment]
Links:
- https://osp-nodeports-1850999--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom.html

- https://osp-nodeports-1850999--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-kuryr.html

- https://osp-nodeports-1850999--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-user.html

- https://osp-nodeports-1850999--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-user-kuryr.html

File: installing/installing_openstack/installing-openstack-user.adoc [hunk 3]
```diff
@@ -59,4 +59,5 @@ include::modules/installation-osp-configuring-floating-ip.adoc[leveloffset=+1]
 
 * xref:../../installing/install_config/customizations.adoc#customizations[Customize your cluster].
 * If necessary, you can
-xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[opt out of remote health reporting].
\ No newline at end of file
+xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[opt out of remote health reporting].
+* If you need to enable external access to node ports, xref:../../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-nodeport.html#nw-using-nodeport_configuring-ingress-cluster-traffic-nodeport[configure ingress cluster traffic by using a NodePort].
```
Comment (bobfuru) [inline installing/installing_openstack/installing-openstack-user.adoc:63]: s/.htmtl/.adoc/

**Review by bobfuru — COMMENTED** [review]
A few xrefs should have `.adoc` extensions and not `.html` but otherwise LGTM.

**Comment by maxwelldb** [comment]
> A few xrefs should have `.adoc` extensions and not `.html` but otherwise LGTM.

Eep.
 
@bobfuru Thanks!

**Pull request merged**